src/notyf/toast-container/toast-container.component.ts

```diff
@@ -0,0 +1,12 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+    selector: 'app-toast-container',
```

Comment:
What if I have a prefix standard for all our components and directives selectors. Something like `notyf-toast-container`, `notyf-toast`, etc. Makes sense to you?